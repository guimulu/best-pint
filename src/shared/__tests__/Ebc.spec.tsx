import * as React from "react";
import { createSnapshot } from "jest-react-fela";

import { Ebc } from "..";

describe("Ebc component green", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<Ebc ebc={9} />);
    expect(snapshot).toMatchSnapshot();
  });
});

describe("Ebc component yellow", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<Ebc ebc={45} />);
    expect(snapshot).toMatchSnapshot();
  });
});

describe("Ebc component red", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<Ebc ebc={50} />);
    expect(snapshot).toMatchSnapshot();
  });
});
