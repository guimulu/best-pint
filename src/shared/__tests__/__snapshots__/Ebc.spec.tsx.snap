// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ebc component green renders correctly 1`] = `
".a {
  background-color: #21ab64;
}
.b {
  padding: 8px;
}
.c {
  position: absolute;
}
.d {
  left: 0;
}
.e {
  top: 72px;
}
.f {
  border-top-left-radius: 8px;
}
.g {
  border-bottom-right-radius: 8px;
}
.h {
  color: #fff;
}
.i::before {
  content: \\"EBC: 9°\\";
}
.j::before {
  font-size: 16px;
}
.k::before {
  font-weight: bold;
}


<div className=a b c d e f g h i j k />;
"
`;

exports[`Ebc component red renders correctly 1`] = `
".a {
  background-color: #d3603a;
}
.b {
  padding: 8px;
}
.c {
  position: absolute;
}
.d {
  left: 0;
}
.e {
  top: 72px;
}
.f {
  border-top-left-radius: 8px;
}
.g {
  border-bottom-right-radius: 8px;
}
.h {
  color: #fff;
}
.i::before {
  content: \\"EBC: 50°\\";
}
.j::before {
  font-size: 16px;
}
.k::before {
  font-weight: bold;
}


<div className=a b c d e f g h i j k />;
"
`;

exports[`Ebc component yellow renders correctly 1`] = `
".a {
  background-color: #e6cf01;
}
.b {
  padding: 8px;
}
.c {
  position: absolute;
}
.d {
  left: 0;
}
.e {
  top: 72px;
}
.f {
  border-top-left-radius: 8px;
}
.g {
  border-bottom-right-radius: 8px;
}
.h {
  color: #fff;
}
.i::before {
  content: \\"EBC: 45°\\";
}
.j::before {
  font-size: 16px;
}
.k::before {
  font-weight: bold;
}


<div className=a b c d e f g h i j k />;
"
`;
